draft-ietf-spice-sd-cwt.md

```diff
@@ -828,9 +833,9 @@ The following completed registration template is provided:
 * Applications that use this media type: TBD
 * Fragment identifier considerations: n/a
 * Additional information:
-      Magic number(s): n/a
-      File extension(s): n/a
-      Macintosh file type code(s): n/a
+     - Magic number(s): n/a
+     - File extension(s): n/a
+     - Macintosh file type code(s): n/a
 * Person & email address to contact for further information:
   Orie Steele, orie@transmute.industries
```

Comment:
We should make this the SPICE WG list.

Reply:
While I think that is generally a good idea, most RFCs (even standards track ones, and even recent ones such as RFC9431) are using a person's name and email address. Is there a document that says we should use WG mailing lists now?  If so, I am happy to change it.

Reply:
Created an [issue](https://github.com/ietf-wg-mediaman/6838bis/issues/9) on [draft-mediaman-6838bis].(https://datatracker.ietf.org/doc/draft-mediaman-6838bis/)

Reply:
Recent documents that I've gotten through publication look like this:

```
   Person and email address to contact for further information:
      See Authors' Addresses section
```

Separately, please see the source of RFC 9458 for a better media type registration template:  https://github.com/ietf-wg-ohai/oblivious-http/blob/main/draft-ietf-ohai-ohttp.md